scripts/peer/peer.py

```diff
@@ -0,0 +1,273 @@
+"""
+Test run this with:
+
+    cd <dir where peer.py file is>
+    analysis-runner \
+        --access-level test \
+        --description 'Test run peer analysis' \
+        --output-dir 'peer/2022-03-21' \
+        --dataset tob-wgs \
+        python3 peer.py \
+        --path-to-cell-files 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/expression_files/'
+"""
+
+
+import os
+import logging
+import click
+import hailtop.batch as hb
+import pandas as pd
+from cpg_utils.hail import remote_tmpdir, output_path
+from google.cloud import storage
+
+PEER_DOCKER = 'australia-southeast1-docker.pkg.dev/cpg-common/images/peer:1.3.2'
+SCORES_PATH = 'gs://cpg-tob-wgs-test/kat/pca/nfe_feb22/v0/scores.json'
+COVARIATES_PATH = 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/covariates_files/covariates.tsv'
+SAMPLE_ID_KEYS_PATH = 'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/metadata/keys_metadata_sheet.csv'
```

Comment:
These constants are good for testing, but it would probably be better to remove them, and always specify the files when you run this script.